src/aiortc/contrib/media.py

```diff
@@ -259,7 +259,7 @@ def __init__(self, file, fps=None, format=None, options={}):
                 self.__streams.append(stream)
             elif stream.type == "video" and not self.__video:
                 if fps is not None and fps < stream.base_rate:
-                    fps_factor = int(float(stream.base_rate) / fps)
+                    fps_factor = round(float(stream.base_rate) / fps)
```

Comment:
Why exactly did you change it? Does it have to do with the wired looking 30 fps and 15 fps?